presto-hive/src/test/java/io/prestosql/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -844,7 +845,7 @@ private static long maxPrecision(int numBytes)
     public void testDecimalBackedByINT32()
             throws Exception
     {
-        for (int precision = 1; precision <= MAX_PRECISION_INT32; precision++) {
+        for (int precision = 4; precision <= MAX_PRECISION_INT32; precision++) {
```

Comment:
why change here?

Reply:
Decimal(3, 0) cannot keep -1000..1000 ints

Reply:
ok, make it separate commit